### DISCUSSION TITLE
It seems startAsync in the BulkMessenger does not work correctly

### DISCUSSION THREAD

**Discussion by ghost** [nutgram/nutgram#305] [Q&A]
my code:

```php
$bot = new Nutgram(config('bot.token'));

$chats = [];
for ($i=0; $i < 50; $i++) { 
    $chats[] = 5811614684;
}

$bot->getBulkMessenger()
    ->setChats($chats)
    ->setText('Hello!')
->startAsync();
```

I run this code from CLI, but it sends the message only once... where is the problem? (I do not receive an error)

**Comment by sergix44** [comment]
For a cli command that only runs the messenger, you must use the sync mode, otherwise the main process is going to terminate before the next alarm.
